sdk/lib/drivers/sound/mmebuddy/devicelist.c

```diff
@@ -51,7 +51,7 @@ FreeSoundDevice(
 {
     SND_ASSERT( SoundDevice );
 
-    SND_TRACE(L"Freeing a SOUND_DEVICE structure");
+    SND_TRACE(L"Freeing a SOUND_DEVICE structure\n");
```

Comment:
```suggestion
    SND_TRACE(L"Freeing SOUND_DEVICE structure\n");
```

Reply:
I agree, but I don't want to tweak wording (in this PR): lines 30 and 77, at least, are affected as well...
(Feel free to follow-up.)

Reply:
Ok, your reasoning to not do that today makes sense. I will merge your PR now. It's trivial enough.